### DISCUSSION TITLE
up-current and query-strings

### DISCUSSION THREAD

**Discussion by apollo13** [unpoly/unpoly#641] [Q&A] [answered]
Hi, I am trying to get the `up-current` class applied when query strings are in use. I have a nav with items like `/users/`, `/groups/` etc. If I click those links `up-current` is applied properly. But those pages are "list views" that allow sorting via a query param (ie `/users/?sort=username`) and in those case unpoly no longer is able to match that to the link in the navigation.

The docs for `up-nav` say that hashes are ignored, would it also be possible to ignore query strings? Maybe via a configuration option? I could probably `up-alias` the links with `/users/*` but that feels kinda weird.

EDIT:// unless I got it wrong `up-alias` doesn't seem to work either.

**Comment by adam12** [comment, marked as answer]
How about two aliases?

`up-alias="/users/ /users/?*"`

See my [glitch](https://glitch.com/edit/#!/tree-important-heaven?path=index.html%3A1%3A0).

**Reply by apollo13** [reply]
I tried `/users/*` but that didn't work. Didn't think of `/users/?*`. Thank you very much :+1: 

Did I miss it in the docs, or is that an area where they could get improved?

**Reply by adam12** [reply]
It's not officially documented. You could add an example to show wildcard matching in a query string if you wanted to submit a PR. I am sure Henning would accept it.

**Reply by apollo13** [reply]
Done in https://github.com/unpoly/unpoly/pull/642
